plugins/azure/virtualmachines/scaleSetAdAuthenticationEnabled.js

```diff
@@ -0,0 +1,60 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Scale Sets Active Directory Authentication Enabled',
```

Comment:
Can we change the title to **Scale Sets AD Authentication Enabled** and file name to **scaleSetAdAuthEnabled**